ci-operator/config/openshift/cincinnati-graph-data/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- crawford
+- lucab
+- steveeJ
+- wking
+- vrutkovs
```

Comment:
I think I should be one of the approvers. WDYT?

Reply:
> I think I should be one of the approvers. WDYT?

Yup.  But you want to get yourself into [here][1] first, and then the release-repo's automatic OWNERS bumper will feed you in here.

[1]: https://github.com/openshift/cincinnati-graph-data/blob/master/OWNERS_ALIASES

Reply:
Got it. Will send a PR there.